docs/sdk-and-tools/indices/rating.md

```diff
@@ -0,0 +1,29 @@
+---
+id: rating
```

Comment:
as discussed, please rename the `id` to `es-index-rating`

Reply:
refactored